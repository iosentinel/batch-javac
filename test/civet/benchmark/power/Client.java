package benchmark.power;

public class Client {
	public static int x(Power pm, Power pa, int base) {
		return pa.raise(pm.raise(base));
	}
}
