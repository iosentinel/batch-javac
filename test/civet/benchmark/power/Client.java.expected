package benchmark.power;

public class Client {
    
    public Client() {
        super();
    }
    
    public static int x(Power pm, Power pa, int base) {
        return pa.raise(pm.raise(base));
    }
    
    public static int x$1002(int base) {
        return benchmark.power.Power.raise$1005(benchmark.power.Power.raise$1003(base));
    }
}